src/core/js/reactHelpers.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Used by babel plugin babel-plugin-transform-react-templates to inject react templates
+ */
+export default function register(name, component) {
+  templates[name] = component;
+};
+
+/**
+ * Storage for react templates
+ */
+export const templates = {};
+
+/**
+ * Convert html strings to react dom, equivalent to handlebars {{{html}}}
+ * @param {string} html
+ */
+export function html(html, ref = null) {
+  let node = html ? HTMLReactParser(html) : '';
```

Comment:
As discussed, please fix when `undefined` is passed.

Reply:
done in https://github.com/adaptlearning/adapt_framework/pull/2829/commits/9b3a7b0dd4551f56532f31aa29456266482af450